scripts/pre_commit_linter.py

```diff
@@ -83,8 +98,20 @@ def _is_javascript_file(filename):
     return filename.endswith('.js')
 
 
+def _is_python_file(filename):
+    """Check if the input filename represents a Python file.
+
+    Args:
+    - filename: str. The name of the file to be checked.
+
+    Returns:
+      bool: True if the filename ends in ".py", and false otherwise
+    """
+    return filename.endswith('.py')
```

Comment:
Cool, in that case, I will be able to completely remove this function and the corresponding js function and use the lambda function in place of these function declarations.